ui/js/controllers/perf/compare.js

```diff
@@ -76,24 +79,30 @@ perf.controller('CompareChooserCtrl', [
             $scope.runCompare = function () {
                 const revisionPromises = [];
                 if ($scope.revisionComparison) {
-                    revisionPromises.push(ThResultSetModel.getResultSetsFromRevision($scope.originalProject.name, $scope.originalRevision).then(
-                        function () {
+                    revisionPromises.push(PushModel.getList({
+                        repo: $scope.originalProject.name,
+                        revision: $scope.originalRevision,
+                    }).then((resp) => {
+                        if (resp.ok) {
                             $scope.originalRevisionError = undefined;
-                        },
-                        function (error) {
-                            $scope.originalRevisionError = error;
-                        },
-                    ));
+                        } else {
+                            $scope.originalRevisionError = resp.statusText;
```

Comment:
Is the resp.statusText assigned here only for the purposes of evaluation (truthy or falsey) or is it being delivered back to the user ultimately (if so, maybe the json response would be more useful)?